tests/test.cpp

```diff
@@ -3222,6 +3222,23 @@ void FixedLengthArrayTest() {
   // Check alignment
   TEST_EQ(0, reinterpret_cast<uintptr_t>(mArStruct->d()) % 8);
   TEST_EQ(0, reinterpret_cast<uintptr_t>(mArStruct->f()) % 8);
+
+  // Check if default constructor set all memory zero
+  const size_t arr_size = sizeof(MyGame::Example::ArrayStruct);
+  char *non_zero_memory = new char[arr_size];
```

Comment:
not that performance matters here, but code would be even simpler with just a local array: `char non_zero_memory[sizeof(MyGame::Example::ArrayStruct)]` ?

Reply:
done
https://github.com/google/flatbuffers/pull/5938/commits/90d7534676bbb621f05bb89fa31b4be15e9e9ab5